src/plugins/bodhi.c

```diff
@@ -271,6 +270,36 @@ static void bodhi_print_errors_from_json(json_object *json)
     return;
 }
 
+/**
+ * Parses only name of rpm package
+ */
+static int parse_pkginfo_name(const char *pkg_full_name, char **name)
```

Comment:
Oh, I see, the function can only parse the name element. The function says it but I didn't get it. How about to add a short description of the algorithm like: "We reverse iterate through the string. When we go over the first dot we parsed release and we can start searching for - that delimits Version from Release. Once we find the Version Release delimiter we can start looking for Name Version delimiter which is again -."

I'm not sure why we need to search for the last dot. It would be nice if you could add a few examples of package names. Or what about to add a link to package name format specification?